frontend/src/components/maps/leaflet/LayerPopup/LayerPopUpContent.test.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { createMemoryHistory } from 'history';
+import { render, cleanup } from '@testing-library/react';
+import { IPopupContentProps, LayerPopupContent } from './LayerPopupContent';
+import { Router } from 'react-router-dom';
+import queryString from 'query-string';
+
+const history = createMemoryHistory();
+jest.mock('hooks/useApi');
+
+const mockLayer: IPopupContentProps = {
+  config: {},
```

Comment:
I hope leaving this empty isn't causing issues?

Reply:
Entirely depends on if your test or the code requires properties to be defined within it.  Otherwise it's entirely acceptable.